src/Mesh/babylon.mesh.vertexData.ts

```diff
@@ -2095,7 +2165,7 @@
             }
 
             // facetPartitioning reinit if needed
-            if (computeFacetPartitioning) {  

```

Comment:
as a general rule, please don't fix spaces in the same change as other changes. it makes merging a lot more difficult. for this code base, I would recommend not fixing the spaces since there are a lot of contributors.